bindings/python/examples/exchange/5_send_amount.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from iota_sdk import Wallet, SyncOptions
+from dotenv import load_dotenv
+import os
+
+# This example uses secrets in environment variables for simplicity which should not be done in production.
+load_dotenv()
+
+# This example sends tokens to an address.
+
+if 'WALLET_DB_PATH' not in os.environ:
+    raise Exception(".env WALLET_DB_PATH is undefined, see .env.example")
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
+if 'EXPLORER_URL' not in os.environ:
+    raise Exception(".env EXPLORER_URL is undefined, see .env.example")
+
+
+wallet = Wallet(os.environ.get('WALLET_DB_PATH'))
+
+account = wallet.get_account('Alice')
+
+wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
+
+# Set syncOnlyMostBasicOutputs to true if not interested in outputs that are timelocked,
```

Comment:
casing of `syncOnly...` should be snake?

Reply:
Done